common/helpers/move/can-edit-move.js

```diff
@@ -0,0 +1,17 @@
+function canEditMove(move, canAccess = () => false) {
```

Comment:
```suggestion
function canEditMove(move, hasPermission = () => false) {
```

I realise this is named to match `req.canAccess` but at first the name was a little confusing to me, I wonder if something like this is more explicit.

Reply:
Yeah, I kept the name of this argument inline with the method that is being passed in. 

I ideally wanted to remove the need to send in this function but currently we have some logic which allows particular edit steps to support specific permissions in case we needed to restrict certain user types to only being able to edit certain bits of information, for example risk/health information but not the date.

If we were to rename this I'd prefer to do this as a separate change as I think we should be consistent in how we reference this method and currently its referred to as `canAccess` in a [number of places](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/search?q=canAccess).